src/marks/area.d.ts

```diff
@@ -4,30 +4,208 @@ import type {Data, MarkOptions, RenderableMark} from "../mark.js";
 import type {BinOptions, BinReducer} from "../transforms/bin.js";
 import type {StackOptions} from "../transforms/stack.js";
 
+/** Options for the area, areaX, and areaY marks. */
 export interface AreaOptions extends MarkOptions, StackOptions, CurveOptions {
+  /**
+   * The vertical position channel for the left side, typically bound to the *x*
```

Comment:
Vertical and horizontal are flipped here, yes? I’ll fix.

Reply:
Ha, yes. (I'm always looking at my hands when I need to figure out left from right.)